Gem/Code/Source/Components/NetworkTeleportComponent.cpp

```diff
@@ -97,15 +97,17 @@ namespace MultiplayerSample
     }
 
     void NetworkTeleportComponentController::TeleportPlayer(
-        const AZ::Vector3& vector, AZ::Entity* entity)
+        [[maybe_unused]] const AZ::Vector3& vector, AZ::Entity* entity)
     {
         if (entity)
         {
             MultiplayerSample::NetworkTeleportCompatibleComponent* teleportable =
                 entity->FindComponent<MultiplayerSample::NetworkTeleportCompatibleComponent>();
             if (teleportable)
             {
+#if AZ_TRAIT_SERVER
                 teleportable->Teleport(vector);
+#endif
```

Comment:
I think this whole function should be wrapped in #AZ_TRAIT_SERVER because the client doesn't appear to do anything except printf when an entity collides that cannot be teleported.  I'm wondering if that comment should be removed to avoid spam

Reply:
I need to poke at this a bit more and make sure no autonomous roles are involved here but I think that's the right call.

Reply:
Looking at this further, I'm not actually sure what's right here. It seems like it could be enabled on Autonomous but I'm not sure if that's intended or not. I think will need further investigation after this change goes in.